app/containers/mainPanel.js

```diff
@@ -189,14 +189,6 @@ class MainPanel extends Component {
     }
   }
 
-  scrollToBottom (force) {
-    if (!force && !this.shouldAutoScroll) return
-    var messagesDiv = document.querySelector('.window__main')
-    if (messagesDiv) {
-      messagesDiv.scrollTop = messagesDiv.scrollHeight
-    }
-  }
-
```

Comment:
not needed anymore?

Reply:
it was a duplicate function! there was already another scrollToBottom function